modules/minimap2/1.0/config/default.yaml

```diff
@@ -0,0 +1,36 @@
+lcr-modules:
+    
+    minimap2:
+
+        # TODO: Update the list of available wildcards, if applicable
```

Comment:
Can you address all the TODOs (here and elsewhere in the module) and then remove these commented lines?